content/python/concepts/math-module/terms/math-atan/math-atan.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'math.atan()'
+Description: 'Returns the inverse tangent of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags: 
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent: 
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`math.atan()`** returns the inverse tangent of a number.
```

Comment:
```suggestion
The **`math.atan()`** returns the inverse tangent of a number.
```
You can make the introduction more brief. It must be at least 160 - 170 characters long

Reply:
You didn't make any changes to my file though. Do you mean I should make it longer and extend it to at least 160-170 characters long? @SaviDahegaonkar